components/clp-py-utils/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "clp-py-utils"
+version = "0.1.0"
+description = "Utilities for other Python packages in CLP."
+authors = ["YScope Inc. <dev@yscope.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8 || ^3.10"
+mariadb = "~1.0.7"
+mysql-connector-python = "~8.0.26"
+pydantic = "^1.8.2"
+python-Levenshtein = "^0.21.1"
+PyYAML = "^6.0.1"
+
+[tool.poetry.group.dev.dependencies]
+black = "^23.9.1"
+docformatter = "^1.7.5"
+mypy = "^1.5.1"
+ruff = "^0.0.291"
```

Comment:
Let's add these dev dependencies in another PR.

Reply:
Ok